backend/controllers/adjudicationHistory.js

```diff
@@ -0,0 +1,110 @@
+const moment = require('moment')
```

Comment:
Is this file supposed to be part of this PR?

Reply:
Nope :D There was a merge conflict when I pulled in from main so looks like it got re added by accident.  Removing, cheers!